templates/inscription.html

```diff
@@ -72,6 +72,8 @@ <h1>Inscription {{ self.number }}</h1>
 %% if let Some(sat) = self.sat {
   <dt>sat</dt>
   <dd><a href=/sat/{{sat}}>{{sat}}</a></dd>
+  <dt>sat name</dt>
+  <dd>{{ sat.name() }}</dd>
```

Comment:
I think it would be cool to also link to `/sat/{sat_name}`. So just add an href and then this can be merged :)